modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinMemoryLeakTest.java

```diff
@@ -232,10 +232,10 @@ public static Collection<Object[]> data() {
                 //MenuBar.class,
 
                 //
-                PasswordField.class,
+                //PasswordField.class,
```

Comment:
This seems an unrelated change.

Reply:
since this is the last PR in the series of skin memory leak fixes, the code block will be removed.